db-server/test/backend/remote.js

```diff
@@ -192,7 +192,7 @@ module.exports = function(cfg, makeServer) {
             assert.equal(!! result[2].isPrimary, false, 'isPrimary is false on thirdEmailRecord email')
             assert.equal(!! result[2].isVerified, false, 'matches secondEmail thirdEmailRecord')
 
-            return client.delThen('/account/' + user.accountId + '/emails/' + secondEmailRecord.email)
+            return client.delThen('/account/' + user.accountId + '/emails/' + emailToHex(secondEmailRecord.email))
```

Comment:
Perfect! But we will also need to update the auth-server to pass an encoded email, similar to https://github.com/mozilla/fxa-auth-server/blob/45ae7b2048e9115590611a55454627d3840119a8/lib/db.js#L1069.

Ref: https://github.com/mozilla/fxa-auth-server/issues/2334

Reply:
Yeah sure i will complete it :)